package.json

```diff
@@ -13,19 +13,19 @@
     "prebuild:dist": "rimraf dist",
     "build:dist": "babel src -d dist",
     "build:docs": "gulp docs",
-    "predeploy:docs": "NODE_ENV=production npm run build:docs",
+    "predeploy:docs": "cross-env NODE_ENV=production npm run build:docs",
     "deploy:docs": "gh-pages -d docs/build -m 'deploy docs [ci skip]'",
     "lint": "eslint .",
     "lint:fix": "npm run lint -- --fix",
     "lint:watch": "watch 'npm run lint --silent' docs gulp src test",
-    "prepublish": "NODE_ENV=production npm run build:dist",
+    "prepublish": "cross-env NODE_ENV=production npm run build:dist",
     "release:major": "ta-script npm/release.sh major",
     "release:minor": "ta-script npm/release.sh minor",
     "release:patch": "ta-script npm/release.sh patch",
     "start": "npm run docs",
     "start:local-modules": "npm run docs -- --local-modules",
     "pretest": "gulp dll",
-    "test": "NODE_ENV=test babel-node $(npm bin)/karma start build/karma.conf.babel.js",
+    "test": "cross-env NODE_ENV=test babel-node node_modules/karma start build/karma.conf.babel.js",
```

Comment:
This should be `node_modules/.bin/karma`.

Details:
When you `npm install` packages that come with executables, they are placed in `node_modules/.bin`.  You can get the absolute path to the npm bin with the command `npm bin`.  NPM also adds this directory to your `$PATH` when you `npm run ...` scripts.  This way you can execute scripts that you have installed.  The `$(npm bin)` syntax is command substitution.  It places the output of that command inline in another command.  So, we were placing the absolute path to the npm bin directory just before `/karma`.